pandas/io/json/normalize.py

```diff
@@ -224,6 +224,30 @@ def _pull_field(js, spec):
         sep = str(sep)
     meta_keys = [sep.join(val) for val in meta]
 
+    def _extract(obj, path, seen_meta, level):
+        recs = _pull_field(obj, path[0])
+
+        # For repeating the metadata later
+        lengths.append(len(recs))
+
+        for val, key in zip(meta, meta_keys):
+            if level + 1 > len(val):
```

Comment:
Could do greater than or equal to comparison rather than incrementing the left hand side and doing a greater than